### PR TITLE
adding samsung

### PR DESCRIPTION
**Pull request by jhaddix**
https://security.samsungmobile.com/securityReporting.smsb

### CODE REVIEW

File: bug-bounty-list/bug-bounty-list.json [hunk 1]
```diff
@@ -6008,6 +6008,16 @@
     "swag": false,
     "hall_of_fame": true,
     "safe_harbor": "partial"
+  },
+   {
+    "program_name": "Samsung",
```
Comment (dantrauner) [inline bug-bounty-list/bug-bounty-list.json]: Is this just Samsung Mobile? we should probably be specific

File: bug-bounty-list/bug-bounty-list.json [hunk 2]
```diff
@@ -6008,6 +6008,16 @@
...
+    "policy_url": "https://security.samsungmobile.com/securityReporting.smsb",
+    "submission_url": "https://security.samsungmobile.com/securityReporting.smsb",
+    "launch_date": "",
+    "cash_reward": true
```
Comment (dantrauner) [inline bug-bounty-list/bug-bounty-list.json]:
```suggestion
    "cash_reward": true,
```

File: bug-bounty-list/bug-bounty-list.json (hunk 1 above)
Comment (jhaddix) [reply bug-bounty-list/bug-bounty-list.json]: yes, sorry should have specified. Thanks

File: bug-bounty-list/bug-bounty-list.json [hunk 3]
```diff
@@ -6008,6 +6008,16 @@
...
+    "swag": false,
+    "hall_of_fame": false
```
Comment (dantrauner) [inline bug-bounty-list/bug-bounty-list.json]:
```suggestion
    "hall_of_fame": false,
```

File: bug-bounty-list/bug-bounty-list.json (hunk 1 above)
Comment (dantrauner) [inline bug-bounty-list/bug-bounty-list.json]:
```suggestion
    "program_name": "Samsung Mobile",
```

**Pull request merged**